syslog/init.sls

```diff
@@ -0,0 +1,20 @@
+# We need rsyslog 8.5+ to monitor plaintext log files using wildcards
+rsyslog:
+  pkgrepo.managed:
+    - ppa: adiscon/v8-stable
+  pkg.latest:
+    - name: rsyslog
+    - refresh: True
+
+# Load the imfile module so we can monitor plaintext log files
+load_imfile:
+  file.prepend:
+    - name: /etc/rsyslog.conf
+    - text: |
+        module(load="imfile" PollingInterval="10")
+
+restart_rsyslog_if_imfile_added:
+  cmd.run:
+    - name: service rsyslog restart
```

Comment:
In `forward_logs/init.sls` and `unattended_upgrades/init.sls`, we use the command 'restart rsyslog'. Might be nice to use the same command name in all places.